### PR TITLE
Public API documentation

### LINKED ISSUE #327: Missing API documentation
Usbguard API is frustruating and contra-productive to use due to the fact that there is little or no documentation for it and API user has to search in the sources of usbguard to find out what each function is doing. This is a huge issue of usability of the usbguard API.

### PR DESCRIPTION
**Pull request by ZoltanFridrich**
Closes #327

### CODE REVIEW

File: src/Library/public/usbguard/KeyValueParser.cpp [hunk 1]
```diff
@@ -35,10 +35,6 @@
 
 namespace usbguard
 {
-  /**
```
Comment (tammar96) [inline src/Library/public/usbguard/KeyValueParser.cpp]: Why did you remove this note?

Comment (ZoltanFridrich) [reply src/Library/public/usbguard/KeyValueParser.cpp]: Because its included in the hpp file documentation and it was only polluting doxygen generated documentation.
